### ISSUE TITLE
ftb-backups2 interferes with mods putting entities out of containers

### ISSUE THREAD

**Issue by pucgenie** [FTBTeam/FTB-Modpack-Issues#4520]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.10.0

### Log Files

https://pste.ch/obunopabaw.pl

### Describe the bug

Multiple times a day our Direwolf20 server crashes due to ftb-backups2 creating a backup just at the same time when a bee leaves a beehive. This leads to a ConcurrentModificationException leading to stopping execution of the productive-bees mod's algorithm leading to the server believing something took too long to execute and shutting down.
This shutdown causes inventories of players to be serialized later than some chunks so that items placed from an inventory into a chunk gets lost (reproducibly).

### Steps to reproduce

Place an AdvancedBeeHive and put one (or more - doesn't matter but enhances probability to reproduce) bees into the world (assigned to this AdvancedBeeHive).
Wait for backups to run.
Eventually the server will crash, first generating a crash report of productive-bees being interrupted unexpectedly and ~60 seconds later the server will shutdown (even when setting  max-tick-time to -1) creating a second crash report.

### Expected behaviour

Backup should run between ticks and not interfere with mods and thus not crash the whole server.

### Additional information

https://github.com/JDKDigital/productive-bees/issues/448 (I proposed a code change to partly mitigate this issue but -> see TL;DR)
https://github.com/CreeperHost/FTB-Backups-2/issues/77

### TL;DR
I don't know how to really synchronize it in a way ftb-backups2 expects mods to do it.

**Comment by Gaz492** [comment]
Could you please confirm if you are still experiencing this issue with the latest pack version?
If you are, would you be able to upload new log files for us so we can take a further look?

**Comment by pucgenie** [comment]
Since https://github.com/JDKDigital/productive-bees/issues/448 was integrated, at least there are no crashes anymore. 
I don't care whether or not bees are duplicated in backups. There are no issues in normal world saves.
I could try to trigger a duplication bug if you want me to - but for now I'm not available the next two~three weeks.
